Kernel_23/doc/Kernel_23/CGAL/Kernel/global_functions.h

```diff
@@ -803,6 +815,15 @@ template <typename Kernel>
 Comparison_result compare_slopes(const CGAL::Segment_2<Kernel> &s1,
 const CGAL::Segment_2<Kernel> &s2);
 
+/*!
+compares the slopes of the segments `(p,q)` and `(r,s)`,
+with `p.z() >= q.z()` and  `r.z() >= s.z()`.
+*/
+template <typename Kernel>
+Comparison_result compare_slopes(const CGAL::Point_3<Kernel> &p,
+                                 const CGAL::Point_3<Kernel> &q,
+                                 const CGAL::Point_3<Kernel> &r,
+                                 const CGAL::Point_3<Kernel> &s);
```

Comment:
I had noticed the singular/plural difference, but that's the way it is in 2D.  In fact it should be singular everywhere.  We could unify that breaking backward compatibility.